src/v2/components/PhotoCard.tsx

```
@@ -151,7 +165,10 @@ const PhotoCard: React.FC<PhotoCardProps> = memo(
         tabIndex={0}
       >
         <ContextMenu>
-          <ContextMenuTrigger className="absolute inset-0">
+          <ContextMenuTrigger
+            className="absolute inset-0"
+            onContextMenu={() => setLastSelectedPhoto(photo)}
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)